README.md

```diff
@@ -1,3 +1,67 @@
-# react-component-variations
+# React-Component-Variations
+**React-Component-Variations** is a library for traversing data from a project's component variations.   
 
-Warning: this project is still very alpha.
\ No newline at end of file
+**What is a Component Variation?**  
+Airbnb uses Storybook (https://github.com/storybooks/storybook) to iteratively test and develop react components, as well as for documentation. We use an abstraction called a *variation* on top of Storybook, which logically represents the specific usages of a more general component inside of our codebase. For example, in [React-Dates](http://airbnb.io/react-dates/?selectedKind=DayPicker&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel), the DayPicker component has variations for when the component has no border, when it has an info panel, when it is vertical, etc. Variations help engineers correctly use the abstractions built into your components by providing examples of concrete usage. 
```

Comment:
I'm not sure that this aligns with @ljharb's vision for this library so I'll let him comment. :)